snuba_sdk/mql/mql.py

```diff
@@ -87,16 +90,26 @@
 """
 )
 
-EXPRESSION_OPERATORS: Mapping[str, ArithmeticOperator] = {
-    "+": ArithmeticOperator.PLUS,
-    "-": ArithmeticOperator.MINUS,
+EXPRESSION_OPERATORS: Mapping[str, str] = {
+    "+": ArithmeticOperator.PLUS.value,
+    "-": ArithmeticOperator.MINUS.value,
 }
 
-TERM_OPERATORS: Mapping[str, ArithmeticOperator] = {
-    "*": ArithmeticOperator.MULTIPLY,
-    "/": ArithmeticOperator.DIVIDE,
+TERM_OPERATORS: Mapping[str, str] = {
+    "*": ArithmeticOperator.MULTIPLY.value,
+    "/": ArithmeticOperator.DIVIDE.value,
 }
 
+AGGREGATION_FUNCTIONS: set[str] = {
```

Comment:
This is missing some aggregations. Offhand I can think of `quantiles` and `uniq` but there are others as well. I would go through all the entity definitions for metrics and generic_metrics to get everything. See for example here: https://github.com/getsentry/snuba/blob/master/snuba/datasets/configuration/generic_metrics/entities/distributions.yaml#L70.

Reply:
Added missing aggregations. What do you think about them living directly on the grammar? It makes separating aggregates, curried aggregates, and arbitrary functions easier.